src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check solver solve status\n",
+    "from pyomo.environ import TerminationCondition\n",
+    "assert results.solver.termination_condition == TerminationCondition.optimal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optimization Results\n",
+    "\n",
+    "Display the results and product specifications"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('total cost = $', value(m.fs.capital_cost) + value(m.fs.operating_cost))\n",
+    "print('operating cost = $', value(m.fs.operating_cost))\n",
+    "print('capital cost = $', value(m.fs.capital_cost))\n",
+    "print()\n",
+    "print('Distillate flowrate = ', value(m.fs.D101.condenser.distillate.flow_mol[0]()), 'mol/s')\n",
+    "print('Benzene purity = ', 100 * value(m.fs.D101.\\\n",
+    "    condenser.distillate.mole_frac_comp[0, \"benzene\"]), '%')\n",
+    "print('Residue flowrate = ', value(m.fs.D101.reboiler.bottoms.flow_mol[0]()), 'mol/s')\n",
+    "print('Toluene purity = ', 100 * value(m.fs.D101.\\\n",
+    "    reboiler.bottoms.mole_frac_comp[0, \"toluene\"]), '%')\n",
+    "print()\n",
+    "print('Conversion = ', 100 * value(m.fs.R101.conversion), '%')\n",
+    "print()\n",
+    "print('Overhead benzene loss in F101 = ', \\\n",
+    "    100 * value(m.fs.F101.vap_outlet.flow_mol_phase_comp[0, \"Vap\", \"benzene\"]) / \\\n",
+    "        value(m.fs.R101.outlet.flow_mol_phase_comp[0, \"Vap\", \"benzene\"]), '%' )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import pytest\n",
+    "assert value(m.fs.operating_cost) == pytest.approx(408342.352, abs=100)\n",
+    "assert value(m.fs.capital_cost) == pytest.approx(29926.759, abs=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Display optimal values for the decision variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Optimal Values')\n",
+    "print()\n",
+    "\n",
+    "print('H101 outlet temperature = ', value(m.fs.H101.outlet.temperature[0]), 'K')\n",
+    "\n",
+    "print()\n",
+    "print('R101 outlet temperature = ', value(m.fs.R101.outlet.temperature[0]), 'K')\n",
+    "\n",
+    "print()\n",
+    "print('F101 outlet temperature = ', value(m.fs.F101.vap_outlet.temperature[0]), 'K')"
```

Comment:
I think we should add a brief discussion/explanation of the results. Things we should note:

* improved product purity and less overhead loss
* decrease in operating cost but increase in capital cost
* observations on key operating conditions (notably any temperatures on bounds).

Reply:
I have added a brief discussion, similar to the one in the paper.
"### Key Takeaways

Observe that optimization was able to reduce the yearly operating cost from \\$427,593 to \\$408,342 (~4.5%). However, the amortized capital cost more than doubled from \\$14,704 to \\$29,927 due to the need to increase the conversion in the reactor (from 75% to 93%) to meet the production and purity constraints. 

Further, observe that the product flow rate and product purity are at their minimum values (0.18 mol/s and 99%, respectively). This is expected as increasing recovery would require more energy and cost to purify the product.


Finally, observe that the operating temperature of the flash (F101) is almost at its lower bound. This helps in minimizing the amount of benzene in the vapor stream leaving the flash."